network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -71,5 +68,54 @@ class ArticlePage(FoundationMetadataPageMixin, Page):
             InlinePanel("authors", label="Author", min_num=0)
         ], heading="Author(s)"),
         StreamFieldPanel('body'),
+        InlinePanel("footnotes", label="Footnotes"),
         FieldPanel('sidebar_summary_title'),
     ]
+
+    @property
+    def next_page(self) -> Union[Page, None]:
```

Comment:
For QA, want to make sure this behavior matches: https://github.com/mozilla/foundation.mozilla.org/issues/5145#issuecomment-683946332

Reply:
I've re-written the functions based on the #5145 comment. I think they're small enough now and different enough that if we wanted to write tests for them it'd be easier as two functions.